diff --git a/node_modules/react-native-reanimated/android/build.gradle b/node_modules/react-native-reanimated/android/build.gradle
index a867b17..5d97519 100644
--- a/node_modules/react-native-reanimated/android/build.gradle
+++ b/node_modules/react-native-reanimated/android/build.gradle
@@ -273,6 +273,7 @@ android {
         cmake {
             version = System.getenv("CMAKE_VERSION") ?: "3.22.1"
             path "CMakeLists.txt"
+            buildStagingDirectory = "${rootProject.buildDir}/reanimated-cxx"
         }
     }
     buildTypes {
